src/main/resources/assets/buildinggadgets/lang/en_us.lang

```diff
@@ -88,3 +88,41 @@ help.gui.buildinggadgets.button.load=Loads data onto a gadget/template from a te
 help.gui.buildinggadgets.button.paste=Pastes data on your computer's clipboard to a pre-existing template, or to a piece of paper, creating a template
 help.gui.buildinggadgets.button.save=Saves the data in the gadget/template to a pre-existing template, or to a piece of paper, creating a template
 
+#config:
```

Comment:
Any specific reason that you've opted for <modid>.name rather than <name>.modid not that yours isn't better it just goes against the current format :P Change them all or stick to the format haha.